src/main/scala/com/danielasfregola/twitter4s/http/oauth/OAuth1Provider.scala

```diff
@@ -80,7 +80,9 @@ private[twitter4s] class OAuth1Provider(consumerToken: ConsumerToken, accessToke
     implicit val ec = materializer.executionContext
     extractRequestBody.map { body =>
       val cleanBody = body.replace("+", "%20")
-      if (cleanBody.nonEmpty) {
+      if (cleanBody.nonEmpty && request.entity
+            .getContentType()
+            .mediaType == MediaTypes.`application/x-www-form-urlencoded`) {
```

Comment:
We need to check this **before merging**, as this could break other endpoints' authentication.

Reply:
May be we return `Map()` if we cannot calculate `Map`